packages/react-components/react-select/src/components/Select/useSelectStyles.ts

```diff
@@ -19,51 +19,22 @@ const iconSizes = {
   large: '24px',
 };
 
-/*
- * TODO: a number of spacing and animation values are shared with react-input.
- * We should probably find a way to share these values between form controls in the theme.
- */
-
+// TODO: This 400 style is not in the typography styles.
+// May need a design change
```

Comment:
yup, IIRC when I looked at Text styles some time back, the weight didn't match up for that one